src/main/java/org/jabref/logic/integrity/BooktitleChecker.java

```diff
@@ -18,6 +19,10 @@ public Optional<String> checkValue(String value) {
             return Optional.of(Localization.lang("booktitle ends with 'conference on'"));
         }
 
+        if (URLUtil.URL_PATTERN.matcher(value).find()) {
+            return Optional.of(Localization.lang("The book title contains a URL"));
```

Comment:
I totally overlooked it.

Consistency is key on tools! -- Now read 4 lines above: That starts with `booktitle`.

```suggestion
            return Optional.of(Localization.lang("booktitle contains a URL"));
```

If you don't like it because of bad English language, change the localiaztion on line 19.

Seeing the title checker, maybe just remove "The book title" and "booktitle" in both strings.